packages/pn-commons/src/components/NotificationDetail/NotificationPaymentRecipient.tsx

```diff
@@ -1,55 +1,129 @@
+import React, { memo, useState, useEffect } from 'react';
+
 import { Download } from '@mui/icons-material/';
-import { Button, Link, Typography } from '@mui/material';
-import React, { Fragment, memo } from 'react';
+import { Box, Button, Link, RadioGroup, Typography } from '@mui/material';
+import { ButtonNaked } from '@pagopa/mui-italia';
 import { getLocalizedOrDefaultLabel } from '../../services/localization.service';
-import { PaymentHistory } from '../../types';
+import {
+  NotificationDetailPayment,
+  PagoPAPaymentHistory,
+  PaymentAttachmentSName,
+  PaymentHistory,
+  PaymentStatus,
+} from '../../types';
+import { formatEurocentToCurrency } from '../../utils';
+import NotificationPaymentPagoPAItem from './NotificationPaymentPagoPAItem';
 
 type Props = {
   loading: boolean;
   payments: Array<PaymentHistory>;
+  onPayClick: (noticeCode?: string, creditorTaxId?: string, amount?: number) => void;
+  handleDownloadAttachamentPagoPA: (name: PaymentAttachmentSName) => void;
+  handleReloadPayment: (payment: Array<PaymentHistory | NotificationDetailPayment>) => void;
 };
 
-const NotificationPaymentRecipient: React.FC<Props> = ({ loading, payments }) => {
-  console.log(payments, loading);
+const NotificationPaymentRecipient: React.FC<Props> = ({
+  loading,
+  payments,
+  onPayClick,
+  handleDownloadAttachamentPagoPA,
+  handleReloadPayment,
+}) => {
+  const [selectedPayment, setSelectedPayment] = useState<PagoPAPaymentHistory | null>(null);
+  const isSinglePayment =
+    payments.length === 1 && payments[0].pagoPA?.status === PaymentStatus.REQUIRED;
```

Comment:
this condition is ok. the second condition is an extra because for status different from REQUIRED we alrready don't show the radio button